src/System.Threading.ThreadPool/tests/ThreadPoolTests.cs

```diff
@@ -30,10 +30,13 @@ public static void ConcurrentInitializeTest()
                 {
                     countdownEvent.Signal();
                     countdownEvent.Wait(ThreadTestHelpers.UnexpectedTimeoutMilliseconds);
-                    Assert.True(ThreadPool.SetMinThreads(processorCount, processorCount));
                 };
 
+            // ThreadPool.SetMinThreads returns false on UapAot
```

Comment:
I think we may simply delete this ThreadPool.SetMinThreads call; the test does not depend on it.